keras/utils/generic_utils.py

```diff
@@ -200,12 +200,28 @@ def func_load(code, defaults=None, closure=None, globs=None):
     code = marshal.loads(code.encode('raw_unicode_escape'))
     if globs is None:
         globs = globals()
+    if closure is not None:
+        closure = func_reconstruct_closure(closure)
     return python_types.FunctionType(code, globs,
                                      name=code.co_name,
                                      argdefs=defaults,
                                      closure=closure)
 
 
+def func_reconstruct_closure(values):
+    """Deserialization helper that reconstructs a closure."""
```

Comment:
Please add `# Arguments` and `# Returns` sections.

Reply:
I can confirm that this is a working solution and the code is reasonable.

I can't edit the changes, but for a simple copy and paste here are the arguments and returns as requested. Any chance we can get this incorporated without additional effort from @soaxelbrooke ?

```
    # Arguments
        closure: closure of the function.

    # Returns
        A reconstructed closure for func_load
```